src/current/v23.1/ui-overview-dashboard.md

```diff
@@ -23,6 +23,14 @@ The **Overview** dashboard displays the following time series graphs. All timest
 
 See the [Statements page]({% link {{ page.version.version }}/ui-statements-page.md %}) for more details on the cluster's SQL statements.
 
+Metrics: `sql.select.count`, `sql.update.count`, `sql.insert.count`, `sql.delete.count`
+
+- [`INSERT ... ON CONFLICT DO UPDATE ...`]({% link {{ page.version.version }}/insert.md %}#on-conflict-clause) statements update the `INSERT` metric (`sql.insert.count`) even when the `DO UPDATE` clause is actually executed. The root of the [abstract syntax tree (AST)]({% link {{ page.version.version }}/architecture/sql-layer.md %}#parsing) is used to increment the metric, not the actual execution details.
```

Comment:
```suggestion
- [`INSERT ... ON CONFLICT DO UPDATE ...`]({% link {{ page.version.version }}/insert.md %}#on-conflict-clause) statements update the `INSERT` metric (`sql.insert.count`) even when the `DO UPDATE` clause is actually executed. The root of the [abstract syntax tree (AST)]({% link {{ page.version.version }}/architecture/sql-layer.md %}#parsing) is used to increment the metric, rather than the actual execution details.
```
I'm wondering whether it might be better to flip this, so "The following SQL commands update the `sql.insert.count` (`INSERT`) metric:

- Line 28
- Line 30

Then line 32 feels more like a note to me than a bullet point.

Reply:
I updated as suggested.